src/utils/__tests__/EventEmitter.unit.ts

```diff
@@ -0,0 +1,23 @@
+
+import { EventHandler } from '../EventsHandler'
+
+describe("emit", () => {
```

Comment:
We need to add a few more tests in this file. There's still at least 1 more use case for emit (you can have multiple subscribers) but also the other methods: `on` and `off`